example-config.yml

```diff
@@ -374,6 +374,30 @@ itinerary:
 #       - WALK
 #       - BICYCLE
 
+### Configure here the carbon output in grams per meter for each mode.
+### massUnits can be 'pound', 'kilogram', 'ounce', or 'gram'
+# co2:
+# . enabled: true
+#   carbonIntensity:
+#     walk: 0
+#     bicycle: 0
+#     car: 0.283
+#     tram: 0.041
+#     subway: 0.041
+#     rail: 0.041
+#     bus: 0.105
+#     ferry: 0.082
+#     cable_car: 0.021
+#     gondola: 0.021
+#     funicular: 0.041
+#     transit: 0.041
+#     leg_switch: 0
+#     airplane: 0.382
+#     micromobility: 0
+#   massUnit: "ounce"
+#   showCo2IfHigher: false // show the relative CO2 even if it's higher than driving
```

Comment:
Is it possible to rename this to `showIfHigher`? Since it's already a child of `co2`